src/features/canvass/components/LocationDialog/pages/HouseholdsPage.tsx

```diff
@@ -68,14 +77,63 @@ const HouseholdsPage: FC<Props> = ({
     return floor0 - floor1;
   });
 
+  useEffect(() => {
+    setSelectedHouseholds([]);
+    setReportMany(false);
+  }, [resetSelection]);
+
```

Comment:
I'm not a fan of this. The fact that a piece of state is named as a verb (`resetSelection`) tips me off that something is weird with the separation of concerns.

Moreover, side effects is a bad thing. It's a necessary evil, but it makes things messier, so all uses of `useEffect()` should be scrutinized, and this is one that I don't think holds up.

As I'm looking at this now, I believe that the reason you've found yourself having to do this is because you have duplicated state. Will elaborate in separate comments.